### ISSUE TITLE
Definiton of PcamPool in global_pool.py

### ISSUE THREAD

**Issue by mitalbert** [jfhealthcare/Chexpert#19]
Hi,

Thanks for the neat repo. Quick question, in global_pool.py the class PcamPool is defined as:

`class PcamPool(nn.Module):

    def __init__(self):
        super(ProbPool, self).__init__()

    def forward(self, feat_map, logit_map):
        assert logit_map is not None

        prob_map = torch.sigmoid(logit_map)
        weight_map = prob_map / prob_map.sum(dim=2, keepdim=True)\
            .sum(dim=3, keepdim=True)
        feat = (feat_map * weight_map).sum(dim=2, keepdim=True)\
            .sum(dim=3, keepdim=True)

        return feat`

should it be `super(PcamPool, self).__init__()?`

**Comment by yil8** [comment]
@mitalbert it's a bug lol. We'll fix it ASAP @yww211 @deadpoppy

**Comment by yil8** [comment]
@mitalbert fixed
